src/meta/src/barrier/mod.rs

```diff
@@ -819,24 +819,20 @@ impl GlobalBarrierManager {
         self.context.tracker.lock().await.abort_all(&err);
         self.checkpoint_control.clear_on_err(&err).await;
 
-        if self.enable_recovery {
```

Comment:
If recovery is not enabled, adhoc recovery can still panic the cluster or lead to data loss in some cases I guess, like using in-memory state store.

Reply:
It is fine, because it's just for test.

Reply:
Can we leave a comment for this?